packages/realm/src/scripts/submit-analytics.ts

```diff
@@ -42,7 +42,6 @@
 //   the other information on.
```

Comment:
As I moved more scripts into TypeScript I took the time to "translate" the submit-analytics script as well.

Reply:
TypeScript first team :smile: